### DISCUSSION TITLE
Replace characters in result items?

### DISCUSSION THREAD

**Discussion by calebmil** [microsoft-search/pnp-modern-search#3943] [Q&A] [answered]
I have a search result where one of my columns (State) is a multi-value field.  One some of the results, there can be tens of values specified,  They all come out as semicolon-separated text that does not wrap.  Overflow on the field appears to default to ellipsis instead of wrapping.

Does anyone know how to:

1. Make the text wrap to a new line?
2. Replace the semicolons with <br> or \r\n or some other mechanism that would make the formatting of these results more readable?

Ideally, the solution would allow me to turn this:

AL;AK;AR;AZ

into this:

AL
AK
AR
AZ

or even better some variant of this:

AL, AK,
AR, AZ

**Comment by kasperbolarsen** [comment, marked as answer]
It sounds like you can use the same approach as I used on the Tags values on a number of layouts. This one might be the best sample, https://github.com/microsoft-search/pnp-modern-search-layouts/tree/main/Results/Handlebars/List%20Tag%20options

**Reply by calebmil** [reply]
That definitely got me what I was looking for.  I changed my split character to ";" instead of "," and referenced the slot names I was using and, BOOM!, magic happened.  You rock @kasperbolarsen!
